### DISCUSSION TITLE
Settings reset on startup

### DISCUSSION THREAD

**Discussion by maxim221** [OpenHantek/OpenHantek6022#326] [Q&A] [answered]
Hello! I'm enjoying the app, thank you!

But I have an issue. When I launch the app as non root user the color and theme settings resets. I can't save it with the graphic interface buttons. Only if I launch the app as root the appearance settings can be saved. Please help-)

**Comment by Ho-Ro** [comment, marked as answer]
Please provide more information about your system and setup. How did you install?
My first idea would be to look at the output of `ls -l ~/.config/OpenHantek` and correct the access rights of the files if necessary.
It is always a bad idea to run user programs as root.

**Reply by maxim221** [reply]
Okay! Thank You for the response! I'll provide all the information needed on Saturday. On my way to another city now

**Comment by maxim221** [comment]
You was absolutely right! Works perfect for me! I've changed the permissions and everything works fine now. Thank You!
